disnake/message.py

```diff
@@ -1882,6 +1897,9 @@ async def reply(
         .. versionchanged:: 2.3
             Added ``fail_if_not_exists`` keyword argument. Defaults to ``True``.
 
+        .. versionchanged:: 2.6
+            Raises :exc:`TypeError` instead of ``InvalidArgument``.
```

Comment:
```suggestion
            Raises :exc:`TypeError` or :exc:`ValueError` instead of ``InvalidArgument``.
```

Reply:
Added in another comment: https://github.com/DisnakeDev/disnake/pull/471#discussion_r886829807